wishlist-backend/products/views.py

```diff
@@ -4,13 +4,14 @@
 from .serializers import ProductSerializer
 
 def get_products(request,category=None):
+
     if category:
         ## When a category is specified in URL
         products = Product.objects.filter(category=category)
     else:
         ## When no category is specified, return all products
         products = Product.objects.all()
-    print(products)
+    # print(products)
```

Comment:
you should delete this.

Reply:
My bad, I'll make sure to double check these in future.